storage/Utils/AppUtil.cc

```diff
@@ -477,6 +477,11 @@ namespace storage
     string
     regex_escape(const string& s, regex::flag_type f)
     {
+	// If the function is needed for any other flags it will have to be checked or
+	// even extended.
+	if (f != regex::extended)
```

Comment:
Would this require to bump the patch version?

Reply:
I do not think so since the function is only internal.